package.json

```diff
@@ -297,7 +296,6 @@
         "sass": "1.34.1",
         "sass-lint": "^1.13.1",
         "sass-loader": "^8.0.2",
-        "sass-variable-parser": "^1.2.2",
```

Comment:
sass-variable-parser has a dependency on node-gyp, which requires python2. we should replace that with a manual process to update variables, since that is done so rarely